### DISCUSSION TITLE
Using a separate config file to programmatically run the parcel, it does not load .parcelrc

### DISCUSSION THREAD

**Discussion by hamzamihaidanielx** [parcel-bundler/parcel#5561] [Q&A]
<img width="808" alt="image" src="https://user-images.githubusercontent.com/12731515/103447983-74506600-4c9b-11eb-9c9b-ea9e4c3baefe.png">
<img width="659" alt="image" src="https://user-images.githubusercontent.com/12731515/103447991-83cfaf00-4c9b-11eb-886f-039f7f41daf8.png">

So basically, I got that separate script to run parcel using `node parcel.js` to bundle all of my assets but the problem is that it does no longer loads whatever is specified into the **.parcelrc** file, basically ignoring it.

If I were to run the CLI command **parcel build**, it does load whatever it is within that **.parcelrc** file.

What am I missing here? The documentation doesn't say much tho 😭

**Comment by mischnic** [comment]
> basically ignoring it.

> The documentation doesn't say much tho

https://v2.parceljs.org/features/parcel-api/ says that https://v2.parceljs.org/plugin-system/api/#InitialParcelOptions lists all possible options. And `filePath` isn't one of them.

As I've written in https://github.com/parcel-bundler/parcel/discussions/5559#discussioncomment-255706, you can specify a filepath to a config file in `config` to override any parcelrc in the project or set `defaultConfig` to change the fallback.
